eagle-external/eagle-cdh/src/aux/eagle-server.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
Thanks for your review. As I mentioned in /scripts/control.sh,"Use eagle.properties instead of eagle.conf, configFormat only support hadoop_xml,properties,gflags", CSD can't generate file end with ".conf", I can't find out better method to replace.